cadwyn/_asts.py

```diff
@@ -55,7 +65,7 @@ def get_fancy_repr(value: Any) -> Any:
         return transform_other(value)
 
 
-def transform_grouped_metadata(value: "annotated_types.GroupedMetadata"):
+def transform_grouped_metadata(value: annotated_types.GroupedMetadata):
```

Comment:
Are we sure that this won't break anything? I would rather stay safe and keep it a string